### DISCUSSION TITLE
Global isIndexing stat

### DISCUSSION THREAD

**Discussion by esmultivac** [meilisearch/product#224] [Feedback & Feature Proposal]
Now that there's only one global update queue, it would make sense to have an isIndexing line when you asks for global stats. That way you can schedule batch updates without looking every index to see if one of them is indexing.

**Comment by curquiza** [comment]
I would add we always recommend checking the [update routes](https://docs.meilisearch.com/reference/api/updates.html#get-an-update-status) instead of the stats route to check if it's indexing or not. The stats routes involve a lot of calculation that can lead to some latency 🙂

**Comment by gmourier** [comment]
Thanks, @esmultivac!

We are also working on the update store and we want to add the possibility to consult the tasks independently of an index. This way, it will be possible to do a `GET` on `/tasks` endpoint and you will be able to quickly fetch the latest one. We'll also rename `update` to `task` because it makes more sense as a resource name. 🤓

**Comment by gmourier** [comment]
I'm locking that discussion since the `/tasks` endpoint gives way to see exactly what is being processed and what happened in the past. We have also introduced filtering capabilities that can be useful for looking at it granularly.

📚 https://docs.meilisearch.com/reference/api/tasks.html#task-object
